osu.Game/Overlays/RankingsOverlay.cs

```diff
@@ -144,100 +137,84 @@ public void ShowSpotlights()
             Show();
         }
 
-        private void loadNewContent()
+        private void selectDisplayToLoad()
```

Comment:
this naming is confusing; there is `selectDisplayToLoad()` and `selectDisplay()`. this should be `changeDisplay()` if anything, and the other methods should be named `createDisplay()` as that's what it actually does.